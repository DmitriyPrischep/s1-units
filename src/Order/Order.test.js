import {fakeOrders} from '../mock/fakeOrders';

import React from 'react';
import Order from './Order';
import {sortTypes} from '../utils/sortOrders';
import {shallow, configure} from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('getDate function', () => {
    it('Fake orders', () => {
        const wrapper = shallow(
          <Order
            order = {fakeOrders} 
          />
        );
        expect(toJson(wrapper)).toMatchSnapshot();
      });
});