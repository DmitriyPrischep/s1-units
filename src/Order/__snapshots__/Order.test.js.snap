// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getDate function Fake orders 1`] = `
<div
  className="Order"
>
  <div
    className="Order-header"
  >
    <span
      className="Order-shop"
    />
    <span>
      NaN undefined, undefined, NaN год
    </span>
  </div>
  <div
    className="Order-items"
  />
</div>
`;
